pkg/resource/statefulset.go

```diff
@@ -356,13 +356,19 @@ func (b StatefulSetBuilder) dbArgs() []string {
 		"--join=" + b.joinStr(),
 		fmt.Sprintf("--advertise-host=$(POD_NAME).%s.%s",
 			b.Cluster.DiscoveryServiceName(), b.Cluster.Namespace()),
-		"--logtostderr=INFO",
 		b.Cluster.SecureMode(),
 		"--http-port=" + fmt.Sprint(*b.Spec().HTTPPort),
 		"--sql-addr=:" + fmt.Sprint(*b.Spec().SQLPort),
 		"--listen-addr=:" + fmt.Sprint(*b.Spec().GRPCPort),
 	}
 
+	if b.Cluster.IsLoggingAPIEnabled() {
+		logConfig, _ := b.Cluster.LoggingConfiguration()
```

Comment:
are you sure it's not necessary to check the error here?

Reply:
We are already validating the logConfig in validate_version.go so if any error would have caught at that point only, either the yaml is not proper or the `cockroach debug check-log-config` is throwing some error